packages/nimble/inst/include/nimble/Utils.h

```diff
@@ -25,9 +25,18 @@ class nimbleTimerClass_ {
   double endNimbleTimer() { return(((double)(clock() - t_start))/CLOCKS_PER_SEC); }
 };
 
```

Comment:
It's pretty common for `ASSERT` macros to use `unlikely(-)` as a hint to the compiler that there will probably not be an error.